src/v17/ast.rs

```diff
@@ -41,10 +41,17 @@ pub enum Sequential {
     SeqCase(Case),
     SeqCall(Expr),
     Event(EventTy, Expr),
-    If(Expr, Vec<Sequential>, Vec<Sequential>),
+    If(SequentialIfElse),
     Assert(Expr, Option<Rc<Sequential>>),
 }
 
+#[derive(Clone, Debug)]
+pub struct SequentialIfElse {
+    pub cond: Expr,
+    pub true_branch: Vec<Sequential>,
+    pub false_branch: Vec<Sequential>,
+}
+
```

Comment:
Hi Sam, 

First of all thanks for the contribution. Overall, it looks great.

What do you think of using the same version of SequentialIfElse we have in [v05](https://github.com/vegaluisjose/vast/blob/master/src/v05/ast.rs#L30-L35). You can leverage the pretty print code. Basically, the struct is

```
#[derive(Clone, Debug, Default)]
pub struct SequentialIfElse {
    pub cond: Option<Expr>,
    pub body: Vec<Sequential>,
    pub elsebr: Option<Rc<Sequential>>,
}
```

I believe having an "optional branch" might be better, because we can define `if` recursively.

Reply:
yeah that sounds good. It'll make the printing logic easier to read